src/components/outline-nav/components/outline-nav-from-headings/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { OutlineHeading, OutlineLinkItem } from 'components/outline-nav/types'
+import { OutlineNavWithActive } from 'components/outline-nav/components'
+import { useMemo } from 'react'
+
+/**
+ * Given a flat array of headings,
+ * Return an array of outline items, for use with OutlineNav components.
+ */
+function buildOutlineItems(headings: OutlineHeading[]): OutlineLinkItem[] {
+	return headings.map((heading: OutlineHeading) => {
+		/* TODO: maybe consider resolving the backticks issue closer to server? */
+		const titleWithoutBackticks = heading.title.replace(/`/g, '')
+		return {
+			title: titleWithoutBackticks,
+			url: `#${heading.slug}`,
```

Comment:
Perhaps we could do all this outline formatting from headings in the server 💭 Then this component could truly be presentational.

Reply:
👍 Added a follow-up task 🎟 [Clean up headings-related OutlineNav uses, to move formatting closer to server](https://app.asana.com/0/0/1204024966433303/f)